### ISSUE TITLE
Add some placeholder images/pages

### ISSUE THREAD

**Issue by tacman** [Spomky-Labs/phpwa#122]
### Description

It'd be nice to have some placeholder images that can be overridden and customized but worked out of the box

For example.
```yaml
            offline_fallback:
               
                image: 'images/offline.svg'
```

We'd need to find some open source or cc-no-attribution files, of course.

Similarly, we could add some default pages for offline at bundle level.  I'll experiment with that in pwa-extra-bundle.

**Comment by Spomky** [comment]
The problem is that if any of the `offline_fallback` option is set, then the offline feature is enabled. Some users may want to keep it disabled.

**Comment by tacman** [comment]
How about adding the assets to the repo, but commenting them out in the yaml.  

In particular, an offline.svg in the bundle would safe the effort of looking one up when the default might be fine.

**Comment by Spomky** [comment]
I prefer to keep the current behavior i.e. opt-in for the offline fallback.

**Comment by tacman** [comment]
Right, it could still be opt-in but provide an asset (offline.svg) with the yaml commented out.
